### DISCUSSION TITLE
Site search box

### DISCUSSION THREAD

**Discussion by Tony-MF** [plausible/analytics#2905] [Anything Else]
Hi,

I have around 50 domains. It would make my life (and similar webmasters) easier if we had a site search box on the dashboard's main page. It could be next to the "Add a website" button. After typing one letter, it would already suggest domains starting with that letter. 

GA have that too. 

Now I have click multiple times to show sites on a next page and only then I can open the desired site.

<img width="1150" alt="image" src="https://user-images.githubusercontent.com/73653258/236507817-5f2fb866-fa17-4937-84da-526456290ff2.png">

Thanks

**Comment by metmarkosaric** [comment]
thanks for the feedback @Tony-MF, would be nice to add! for now you can use the "?per_page=" parameter like this: https://plausible.io/sites?per_page=1000. with that link, all your sites will be listed on one page without pagination. you can then use the search functionality in your browser (press cmd+f on a Mac or ctrl+f on Linux and Windows) to instantly find the site you're looking for

**Comment by metmarkosaric** [comment]
**Update:** We’ve made some improvements to the “My Sites” page:

* You can now search for any of your sites
* There’s a graph for an overview of the traffic 
* And a comparison to the previous period so you can see the trend

Have fun exploring!

**Reply by Tony-MF** [reply]
Great news, thanks! Can't wait for it to reach the self-hosted version.

**Reply by chrisdeeming** [reply]
Ah so @metmarkosaric just so I understand, this has been essentially added upstream, but we should see it in the self-hosted version at the next release? Do you have a rough ETA for that?

**Reply by ruslandoga** [reply]
@chrisdeeming 👋 

It's available in the [release candidates](https://github.com/plausible/analytics/releases) for v2.1.0. Here's the discussion for the latest one: https://github.com/plausible/analytics/discussions/4084

**Reply by chrisdeeming** [reply]
Thanks @ruslandoga! 🫶🏻
